protocol-definitions/ReplicatedMap.yaml

```diff
@@ -852,3 +852,75 @@ methods:
           doc: |
             partition id of the entries
     response: {}
+  - id: 20
+    name: fetchEntryViews
+    since: 2.7
+    doc: |
+      Allows iteration over ReplicatedMapEntryViewHolder objects. The changes happened during the iteration may not be included in the iterated EntryViews. This method
+      will throw an exception if there is no replicated record store with the given replicated map name and partition id. This method will consume some memory in the 
+      member with the default timeout of 300 seconds that is reset after each fetch. Sending endEntryViewIteration or timing out will release the resource.
+    request:
+      retryable: true
+      partitionIdentifier: -1
+      params:
+        - name: name
+          type: String
+          nullable: false
+          since: 2.7
+          doc: |
+            Name of the ReplicatedMap
+        - name: cursorId
+          type: UUID
+          nullable: true
+          since: 2.7
+          doc: |
+            The identifier of the last fetched page. Initially null and then returned by this API. 
```

Comment:
How do you guarantee the uniqueness of the UUID? Let's assume I am doing many `fetchEntryViews ` calls(may be million).

Reply:
It's a UUID, it should be a non-issue.